platform/commonUI/edit/test/actions/SaveAsActionSpec.js

```diff
@@ -50,7 +50,7 @@ define(
                         catch: function (callback) {
                             return mockPromise(callback(value));
                         }
-                    }   ;
+                    } ;
```

Comment:
Why is there still a single space before the semicolon? Shouldn't this be a violation?